kernel/syscall/execute.rs

```diff
@@ -212,7 +213,7 @@ pub fn execute(mut args: Vec<String>) -> Result<usize> {
 
                     // debugln!("{}: {}: execute {}", context.pid, context.name, url.string);
 
-                    context.name = url.to_string();
+                    context.name = url.scheme().to_owned().into();
```

Comment:
This looks odd, this will probably break the context list `ps` because all entries will be named "file"